lightwood/data_schemas/predictor_config.py

```diff
@@ -20,11 +20,15 @@
 })
 
 mixer_schema = Schema({
-    'class': object,
+    Optional('class'): object,
     Optional('attrs'): dict,
-    Optional('mixer_graph'): [mixer_graph_schema]
+    Optional('deterministic', default=True): bool,
+    Optional('selfaware', default=True): bool
```

Comment:
should not this be in the attrs?

Reply:
Hmh, weren't the attributes mainly a placeholder ?

As in, if we know the value of some of these attributes, as long as we leave them as optional (aka some mixers might not have them) is it no better to make them explicit like this ?